<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniWindow</class>
 <widget class="QDialog" name="MiniWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnMinimize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-image: url(:/icon/resources/icon/minimize.png);
	background-color: rgba(255, 255, 255,0);
	border-style:outset;
	border-color:rgba(255,255,255,0);
	border-radius:4px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255,100);
}

QPushButton:pressed
{
	background-color: rgba(95, 95, 95,100);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-image: url(:/icon/resources/icon/close.png);
	background-color: rgba(255, 255, 255,0);
	border-style:outset;
	border-color:rgba(255,255,255,0);
	border-radius:4px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255,100);
}

QPushButton:pressed
{
	background-color: rgba(95, 95, 95,100);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreSong">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-image: url(:/icon/resources/icon/playPreSong.png);
	background-color: rgba(255, 255, 255,0);
	border-style:outset;
	border-color:rgba(255,255,255,0);
	border-radius:4px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255,100);
}

QPushButton:pressed
{
	background-color: rgba(95, 95, 95,100);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlayOrPause">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}
QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="songSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/image/image/圆形 (9).png);
  margin: -6 0px; 
    }
      
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextSong">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-image: url(:/icon/resources/icon/playNextSong.png);
	background-color: rgba(255, 255, 255,0);
	border-style:outset;
	border-color:rgba(255,255,255,0);
	border-radius:4px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255,100);
}

QPushButton:pressed
{
	background-color: rgba(95, 95, 95,100);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNormalStyle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-image: url(:/icon/resources/icon/PureTouchIcon3.ico);
	background-color: rgba(255, 255, 255,0);
	border-style:outset;
	border-color:rgba(255,255,255,0);
	border-radius:4px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255,100);
}

QPushButton:pressed
{
	background-color: rgba(95, 95, 95,100);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
